phi/tools/fal.py

```diff
@@ -0,0 +1,62 @@
+"""
+pip install fal-client
+"""
+
+from os import getenv
+from typing import Optional
+
+from phi.agent import Agent
+from phi.tools import Toolkit
+from phi.utils.log import logger
+
+try:
+    import fal_client  # type: ignore
+except ImportError:
+    raise ImportError("`fal_client` not installed. Please install using `pip install fal-client`")
+
+
+class Fal(Toolkit):
```

Comment:
Our naming seems to be inconsitent. Sometimes it is `ThingTools` or `ThingToolkit` or just `Thing`

Reply:
I think we should stick with Tool or ToolKit, I have changed it to toolkit

Reply:
Thanks Yash, I'll also move to tools